app/models/color_scheme.rb

```diff
@@ -89,12 +101,14 @@ class ColorScheme < ActiveRecord::Base
       "quaternary" => "7e566a",
       "header_background" => "a97189",
       "header_primary" => "d9b2bb",
-      "highlight" => "6c3e63",
+      "highlight" => "bd36a3",
+      "selected" => "2a1620",
+      "hover" => "331b27",
       "danger" => "6c3e63",
       "success" => "d9b2bb",
       "love" => "d9b2bb",
     },
-    WCAG: {
+    "WCAG Light": {
```

Comment:
I don't think we need to change the name here, this is an internal name, to clients and users we should a translated label from https://github.com/discourse/discourse/blob/main/config/locales/server.en.yml#L4240 

(And I suspect changing this might require renames in other places.)

Reply:
Oh weird, I thought I had already reverted that, wasn't meant to be here!